key_test.go

```diff
@@ -75,14 +85,14 @@ func TestMarshal(t *testing.T) {
 	}
 	_ = tls_key
 
-	der, err := key.MarshalPKCS1PrivateKeyDER()
+	pkcs1Der, err := key.MarshalPKCS1PrivateKeyDER()
 	if err != nil {
 		t.Fatal(err)
 	}
 	tls_der := x509.MarshalPKCS1PrivateKey(tls_key)
-	if !bytes.Equal(der, tls_der) {
+	if !bytes.Equal(pkcs1Der, tls_der) {
```

Comment:
I think you can use `require.Equal` here -  it prints the two values upon failure as well

Reply:
Agreed, but require is not currently used and this matches the current codebase style